EasyPost/EasyPost.csproj

```diff
@@ -92,6 +92,7 @@
     <Compile Include="Pickup.cs" />
     <Compile Include="PostageLabel.cs" />
     <Compile Include="Properties\AssemblyInfo.cs" />
+    <Compile Include="..\EasyPost\VersionInfo.cs" Link="VersionInfo.cs" />
```

Comment:
Why are there 2 references to `VersionInfo.cs` here? One that goes back up a directory and one for the file in this directory? Should there only be 1 or is there a reason for the second?